### ISSUE TITLE
Staffeli example

### ISSUE THREAD

**Issue by alexpanter** [DIKU-EDU/staffeli#16]
The guide at [http://staffeli.readthedocs.io/en/latest/examples.html#list-students-who-did-not-submit](url) :

- `cat students/.staffeli.yml | grep sis_login_id`
_change_to_ 
`cat students/.staffeli.yml | grep login_id`
(changes in canvas ?)

**Comment by nqpz** [comment]
Right, the Canvas team deprecated `sis_login_id`.  Good catch!  Can you make a pull request with a fix?

**Comment by alexpanter** [comment]
I can, except I don't know which repository contains the html document..

**Comment by nqpz** [comment]
The magic is that this exact repository contains all the docs; see https://github.com/DIKU-EDU/staffeli/blob/master/docs/source/examples.rst for this example.

**Comment by alexpanter** [comment]
It has been updated. As for the guide in the link you posted above, I just created a bash script that will do all the complicated terminal commands like `cut`, `grep`, `perl` etc. I will make a pull request with it, so you can check it out - it could be useful to further automate the process.
